acceptance/apps_test.go

```diff
@@ -890,6 +890,35 @@ configuration:
 			)
 		})
 
+		It("deletes a batch of applications", func() {
+			app1 := catalog.NewAppName()
+			env.MakeContainerImageApp(app1, 1, containerImageURL)
+			app2 := catalog.NewAppName()
+			env.MakeContainerImageApp(app2, 1, containerImageURL)
+
+			var out string
+			var err error
+			Eventually(func() error {
+				out, err = env.Epinio("", "app", "show", app1)
+				return err
+			}, "1m").ShouldNot(HaveOccurred(), out)
```

Comment:
Do we have a helper function doing this kind of check for apps ? If not, should we create it ?
Does `MakeContainerImageApp` perform this check ? Should the check go there ?

Reply:
You are right, `MakeContainerImageApp` already checks this. I'm removing the extra checks from here.